src/edgekv/ekv-handler.ts

```diff
@@ -3,15 +3,21 @@ import * as cliUtils from '../utils/cli-utils';
 import * as response from './ekv-response';
 import * as ekvhelper from './ekv-helper';
 
-export async function listNameSpaces(environment: string) {
+export async function listNameSpaces(environment: string, detail) {
```

Comment:
very minor but you can change this to details since you call this function with param: options.details and to keep consistency.

Reply:
updated